BUILD.md

```diff
@@ -0,0 +1,39 @@
+BUILD INSTRUCTIONS
+==================
```

Comment:
should we not just update `README.md`, rather than have build instructions in multiple places?

Reply:
or reference this from README